web/html/src/manager/minion/packages/package-states.renderer.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import PackageStates from './package-states';
 import SpaRenderer from "core/spa/spa-renderer";
+import {MessagesContainer} from "components/toastr/toastr";
 
 export const renderer = (id, {serverId}) => {
   SpaRenderer.renderNavigationReact(
-    <PackageStates
-      serverId={serverId}
-    />,
+    <React.Fragment>
+      <MessagesContainer/>
+      <PackageStates
+        serverId={serverId}
+      />
+    </React.Fragment>,
```

Comment:
Cosmetic nitpicking, but you can replace the verbose `<React.Fragment></React.Fragment>` by the much shorter `<></>`